Applications/LLMEval/ContentView.swift

```diff
@@ -198,7 +203,7 @@ class LLMEvaluator {
             let prompt = modelConfiguration.prepare(prompt: prompt)
             let promptTokens = MLXArray(tokenizer.encode(text: prompt))
 
-            let initTime = Date()
+            var initTime = Date()
```

Comment:
I guess you don't really need to initialize it here?

Reply:
It is used on the line below to show the time spent preparing, so this is still needed.